lib/ansible/plugins/lookup/cf_output.py

```diff
@@ -0,0 +1,115 @@
+# (c) 2015, Stephen Granger <stephen(at)trinimbus.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.errors import AnsibleError
+from ansible.plugins.lookup import LookupBase
+import boto.ec2
+import boto.cloudformation
+import os
+import sys
+import time
+import pickle
+
+# region/stack/param
+class LookupModule(LookupBase):
+    def __init__(self, basedir=None, **kwargs):
+        self.basedir = basedir
+        self.cache_dir = os.path.join(os.environ['HOME'],'.stack_outputs')
+        self.cache_time = 60
+
+    def check_cache(self, file):
+        now = int(time.time())
+        data = ''
+        if os.path.isfile(file):
+            # check time stamp of file
+            if ( now - int(os.path.getmtime(file)) ) < self.cache_time:
+                fh = open(file, 'r')
+                data = pickle.load(fh)
+
+        return data
+
+    def get_regions(self):
+        regions_cache = os.path.join(self.cache_dir, 'regions')
+        regions = self.check_cache(regions_cache)
+        if regions:
```

Comment:
Rather than passing if something exist, you could check for it not existing
if not regions:
   do something